tests/HtmlTest.php

```diff
@@ -609,4 +609,28 @@ public function testScript() {
          unlink(GLPI_TMP_DIR . '/' . $fake_file);
       }
    }
+
+   /**
+    * @covers Html::getCopyrightMessage()
+    */
+   public function manageRefreshPage() {
+      $message = Html::manageRefreshPage();
+
+      $text ="<script type=\"text/javascript\">\n";
+      $text.="setInterval(\"window.location.reload()\",".
+            (60000 * $_SESSION['glpirefresh_ticket_list']).");\n";
```

Comment:
Travis refused to start for this PR... But I guess we will have a notice here, since there is nothing in the session. You'll probably have to set the value in the session at the beginning of the test.